packages/headless/src/features/analytics/analytics-utils.ts

```diff
@@ -145,9 +145,6 @@ export type ProductRecommendationAction =
 export type ProductListingAction =
   PreparableAnalyticsAction<StateNeededByProductListingAnalyticsProvider>;
 
-export type ProductListingV2Action =
```

Comment:
We can also delete `makeCommerceAnalyticsAction`; it was only used in facet-set-product-listing-v2-analytics-actions.ts, which you're deleting in this PR.